src/modal/modal.scss

```diff
@@ -67,11 +69,77 @@
         margin-right: $iui-s;
       }
     }
+
+    .iui-modal-content {
+      flex-grow: 2;
```

Comment:
```suggestion
      flex-grow: 2;
      margin: 0 -#{$iui-m};
      padding: 0 $iui-m;
```
Can we add this margin & padding so that the scrollbars aren't indented?

**Currently:**
![Screen Shot 2022-03-09 at 2 23 44 PM](https://user-images.githubusercontent.com/849817/157516322-1eac9fc1-a5e2-4ef8-a033-d1c527cd4c67.png)

**With this change:**
![Screen Shot 2022-03-09 at 2 22 59 PM](https://user-images.githubusercontent.com/849817/157516208-657c127d-c5dc-4a6b-b4a3-b613a2c67ef9.png)

Reply:
Done